Integrations/src/main/java/io/deephaven/integrations/python/PythonDeephavenSession.java

```diff
@@ -144,13 +143,6 @@ public Thread newThread(@NotNull Runnable r) {
         }
     }
 
-    @Override
-    @VisibleForTesting
-    public QueryScope newQueryScope() {
-        // depend on the GIL instead of local synchronization
```

Comment:
Note that this comment was never accurate, even in DHE - python can release the GIL whenever it wants to let another thread advance.